### DISCUSSION TITLE
Welcome to Final-Project Discussions!

### DISCUSSION THREAD

**Discussion by paperboy6891** [paperboy6891/Final-Project#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arhmAli** [comment]
Hello im arham khan im a coder on his journey to become something good for the community and myself.
